### ISSUE TITLE
List.Item height

### ISSUE THREAD

**Issue by agat** [callstack/react-native-paper#914]
## Environment

Latest stable version of paper.

## Description

Based on Material Design specs list item height must be:

- With title: 48px
- With title and icon: 56px
- With title and description: 72px

https://material.io/design/components/lists.html#specs

But now `List.Item` component height are:

- With title: 54px
- With title and icon: 72px
- With title and description: 72px

## Reproducible Demo

````
<List.Item
    title="Trello"
    left={props => <List.Icon icon="folder" />}
    description="Project 1, Project 2"
/>
<List.Item
    title="Trello"
    left={props => <List.Icon icon="folder" />}
/>
<List.Item
    title="Trello"
/>
````

![Android Emulator - Pixel_API_25:5554 2019-03-12 13-44-04](https://user-images.githubusercontent.com/427919/54197585-fcfd8100-44cc-11e9-941f-47b44d2c3363.png)

**Comment by ocarreterom** [comment]
Yes, List.Item lacks some variants.

**Comment by ferrannp** [comment]
@agat wanna make a PR?

**Comment by Taym95** [comment]
If no one working on this issue, I would like to work on it.

**Comment by agat** [comment]
There is a desire to try to do it, but there is no time in the coming days.

**Comment by Trancever** [comment]
@Taym95 Fell free to submit PR :)

**Comment by smkhalsa** [comment]
@Taym95 Have you had a chance to take a stab at this?

**Comment by mitulsavani** [comment]
I would like to give a shot on this issue too! 
Can somebody quickly tell me how I can test the component after making changes?

**Comment by Trancever** [comment]
@mitulsavani There is an Example app in `Example` directory. For more informations check [Contributing Guide](https://github.com/callstack/react-native-paper/blob/master/CONTRIBUTING.md)

**Comment by Trancever** [comment]
Fix is included into [3.0.0-alpha.0](https://github.com/callstack/react-native-paper/releases/tag/v3.0.0-alpha.0).
